apps/ui-kit/lib/components/sql-text-editor/v2/SqlTextEditor.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="BksUiKit BksTextEditor BksSqlTextEditor" ref="editor"></div>
```

Comment:
[nitpick] really with css you should use `bks-ui-kit` vs `BksUiKit`

Reply:
This was inspired by Material UI honestly lol. They name their components like `MuiBase-root`.